doc/pages/blurbs.json

```diff
@@ -1,505 +1,514 @@
 [{
-    "name": "Julia Nguyen",
```

Comment:
Can you keep the previous indentation for this file? It should be 2 space instead of 4.

Reply:
Oops 🙈Didn't notice my code editor add those spaces!